bigflow/build/dataflow/depscheck.py

```diff
@@ -0,0 +1,116 @@
+import re
```

Comment:
Maybe `dependency_checker.py`?

Reply:
IMO it is a too long name for python module.

Reply:
`dependency_checker.py` is ok in my opinion

Reply:
Ok, if you both think it should be `dependency_checker`. I've renamed this module.